src/components/number-input/number-input.ts

```diff
@@ -229,6 +276,16 @@ export default class BXNumberInput extends BXInput {
             ${this.validityMessage}
```

Comment:
Should we hide this section if we should out-of-range validity message?

Reply:
I'm not sure ... I was debating that too ... It may make sense to show it, since it's still invalid, but since the range error is a specific type of invalid maybe it would be better to make it clearer that it's just the range that's an issue?

Reply:
Yeah, probably if the validity message is something like "The value should be less than 100" user knows that user did something wrong as well as how to fix it...?

Reply:
Yeah ... So, maybe for now we'll exclude the standard message since that seems like a sensible default